metadata.yaml

```diff
@@ -0,0 +1,366 @@
+id: "terraform-ibm-modules/function/ibm/1.0.0"
+name: "function"
+namespace: "terraform-ibm-modules"
+version: "1.0.0"
+providers: ["ibm"]
+description: "Terraform modules to create cloud function resources"
+source: "https://github.com/terraform-ibm-modules/terraform-ibm-function"
+owner: "umarali.nagoor@in.ibm.com"
+published_at: "2021-11-17T14:54:28.979568Z"
+registry: "https://registry.terraform.io/modules/terraform-ibm-modules/function/ibm/latest"
+alias: "function"
+tags: ["function", "ibm-cloud", "ibm-cloud-function"]
+root:
+  - path: "github.com/terraform-ibm-modules/terraform-ibm-function"
+    name: "function"
+    provider_dependencies:
+      - name: "ibm"
+        namespace: "IBM-Cloud"
+        source: "IBM-Cloud/ibm"
+        version: ""
+###################################
+# input_variables
+###################################
+    inputs:
```

Comment:
Looks like we have not captured information about is_new_rule

Reply:
Fixed